### PR TITLE
MotoROS v1.8.2

### PR DESCRIPTION
**Pull request by ted-miller**
- Increase starting pulse deviation threshold.
- Add 4 hour limit to a single trajectory timestamp.
- Add message to retrieve DH parameters of arm.

### CODE REVIEW

**Comment by ted-miller** [comment]
My robot is tied up with other testing right now. 

@EricMarcil or @gavanderhoorn 
If one of you is available next week to test this, I would appreciate the help. Otherwise, I will perform testing on Nov 26.

**Comment by EricMarcil** [comment]
@gavanderhoorn : I've started doing some testing.  I've found a mistake that prevent the message from sending a proper response.  I'll review with Ted and let you know once we have that part working.

File: motoman_driver/MotoPlus/MpRosAllControllers.vcxproj
Comment (gavanderhoorn) [inline motoman_driver/MotoPlus/MpRosAllControllers.vcxproj:116]: Is this correct? Seems like a strange name for the output binary?

Comment (ted-miller) [reply motoman_driver/MotoPlus/MpRosAllControllers.vcxproj:116]: Yes, this is correct. That field doesn't actually control the file name of the output binary (this isn't a true makefile project). This trick allows Visual Studio to launch the remote-upload utility when you build and attempt to "run" the application. So, after you build, you can easily push the updated binary into the robot controller without booting into maintenance mode. (This only works on YRC.)

**Review by gavanderhoorn — CHANGES_REQUESTED** [review]
I cannot review this properly without my hw working, but I've got confirmed reports that it works and don't want to keep this from being merged any longer.

@ted-miller: could you please remove the `*.vcxprojAssemblyReference.cache` from the PR? I tried to do that for you, but it appears you've unchecked "allow edits from maintainers".

**Comment by ted-miller** [comment]
I have made the change. BTW, "allow edits from maintainers" is actually checked.

**Comment by gavanderhoorn** [comment]
> I have made the change.

thanks.

> BTW, "allow edits from maintainers" is actually checked.

hm, strange. Just tried to push an update to `.gitignore` but I'm seeing this:

```
 ! [remote rejected] tedmiller_kinetic-devel -> tedmiller_kinetic-devel (permission denied)
```

perhaps it's because you have created the PR from a branch you've protected?

**Review by gavanderhoorn — APPROVED** [review]
:+1:-ing these changes.

Further fixes and improvements will be made in follow-up PRs.

**Comment by gavanderhoorn** [comment]
We'll wait for Travis and then merge.

**Comment by gavanderhoorn** [comment]
Thanks for the PR @ted-miller and thanks for testing @EricMarcil.

**Comment by gavanderhoorn** [comment]
Updated wiki links [here](http://wiki.ros.org/action/info/motoman_driver/Tutorials/indigo/InstallServer?action=diff&rev2=36&rev1=35).

**Pull request merged**
